src/main/java/net/dv8tion/jda/internal/managers/GuildManagerImpl.java

```diff
@@ -329,6 +334,40 @@ private GuildManager updateFeatures(Collection<String> changed, Consumer<String>
         return this;
     }
 
+    @Nonnull
+    @Override
+    public GuildManager setSystemChannelFlags(@Nonnull Collection<SystemChannelFlag> flags)
+    {
+        Checks.noneNull(flags, "System channel flag");
+        this.systemChannelFlags = flags;
```

Comment:
Make a copy into an enum set, there is a method in Helpers for it, in case the collection is empty